src/resources/corp-info.ts

```diff
@@ -0,0 +1,294 @@
+    colinInd: true,
+    corpClass: CorpClassE.BC,
+    shortDesc: 'CO 1897',
+    fullDesc: 'CO 1897'
+  }, {
+    corpTypeCd: CorpTypeCdE.REGISTRATION,
+    colinInd: true,
+    corpClass: CorpClassE.XPRO,
+    shortDesc: 'REGISTRATION',
+    fullDesc: 'Registration (Extra-pro)'
+  }, {
+    corpTypeCd: CorpTypeCdE.RAILWAYS,
+    colinInd: false,
+    corpClass: CorpClassE.OT,
+    shortDesc: 'RAILWAYS',
+    fullDesc: 'Railways'
+  }, {
+    corpTypeCd: CorpTypeCdE.SOCIETY,
+    colinInd: true,
+    corpClass: CorpClassE.SOC,
+    shortDesc: 'SOCIETY',
+    fullDesc: 'BC Society'
+  }, {
+    corpTypeCd: CorpTypeCdE.SOCIETY_BRANCH,
+    colinInd: false,
+    corpClass: CorpClassE.OT,
+    shortDesc: 'SOCIETY BRANCH',
+    fullDesc: 'BC Society'
+  }, {
+    corpTypeCd: CorpTypeCdE.CONT_IN_SOCIETY,
+    colinInd: true,
+    corpClass: CorpClassE.SOC,
+    shortDesc: 'CONT IN SOCIETY',
+    fullDesc: 'BC Society'
+  }, {
+    corpTypeCd: CorpTypeCdE.SOLE_PROP,
+    colinInd: true,
+    corpClass: CorpClassE.FIRM,
+    shortDesc: 'SOLE PROP',
+    fullDesc: 'BC Sole Proprietorship'
+  }, {
+    corpTypeCd: CorpTypeCdE.TRUST,
+    colinInd: false,
+    corpClass: CorpClassE.OT,
+    shortDesc: 'TRUST',
+    fullDesc: 'Trust'
+  }, {
+    corpTypeCd: CorpTypeCdE.TRAMWAYS,
+    colinInd: false,
+    corpClass: CorpClassE.OT,
+    shortDesc: 'TRAMWAYS',
+    fullDesc: 'Tramways'
+  }, {
+    corpTypeCd: CorpTypeCdE.ULC_CO_1860,
+    colinInd: true,
+    corpClass: CorpClassE.BC,
+    shortDesc: 'ULC CO 1860',
+    fullDesc: 'ULC CO 1860'
+  }, {
+    corpTypeCd: CorpTypeCdE.ULC_CO_1862,
+    colinInd: true,
+    corpClass: CorpClassE.BC,
+    shortDesc: 'ULC CO 1862',
+    fullDesc: 'ULC CO 1862'
+  }, {
+    corpTypeCd: CorpTypeCdE.ULC_CO_1878,
+    colinInd: true,
+    corpClass: CorpClassE.BC,
+    shortDesc: 'ULC CO 1878',
+    fullDesc: 'ULC CO 1878'
+  }, {
+    corpTypeCd: CorpTypeCdE.ULC_CO_1890,
+    colinInd: true,
+    corpClass: CorpClassE.BC,
+    shortDesc: 'ULC CO 1890',
+    fullDesc: 'ULC CO 1890'
+  }, {
+    corpTypeCd: CorpTypeCdE.ULC_CO_1897,
+    colinInd: true,
+    corpClass: CorpClassE.BC,
+    shortDesc: 'ULC CO 1897',
+    fullDesc: 'ULC CO 1897'
+  }, {
+    corpTypeCd: CorpTypeCdE.XPRO_COOP,
+    colinInd: false,
+    corpClass: CorpClassE.OT,
+    shortDesc: 'XPRO COOP',
+    fullDesc: 'Extraprovincial Cooperative Association'
+  }, {
+    corpTypeCd: CorpTypeCdE.XPRO_LL_PARTNR,
+    colinInd: true,
+    corpClass: CorpClassE.FIRM,
+    shortDesc: 'XPRO LL PARTNR',
+    fullDesc: 'Extrapro Limited Liaility Partnership'
+  }, {
+    corpTypeCd: CorpTypeCdE.XPRO_LIM_PARTNR,
+    colinInd: true,
+    corpClass: CorpClassE.FIRM,
+    shortDesc: 'XPRO LIM PARTNR',
+    fullDesc: 'Extraprovincial Limited Partnership'
+  }, {
+    corpTypeCd: CorpTypeCdE.XPRO_SOCIETY,
+    colinInd: true,
+    corpClass: CorpClassE.SOC,
+    shortDesc: 'XPRO SOCIETY',
+    fullDesc: 'Extraprovincial Society'
+  }
+]
```

Comment:
Haha, hope this was copy pastable from somewhere :D

Reply:
yes haha